ratingTypes.ts

```diff
@@ -20,6 +20,37 @@ export interface Rating {
     dateCreated: Date;
     dateEdited?: Date;
 }
+
+export type PlaceWithAccesibilityData = Place & {
+	accessibilityData?: {
+		_id: string;
+        guideDogAvg: number;
+		isMenuAccessibleAvg: number;
+		noiseLevelAvg: number;
+		lightingAvg: number;
+		isStaffHelpfulAvg: number;
+		isBathroomOnEntranceFloorAvg: number;
+		isContactlessPaymentOfferedAvg: number;
+		isStairsRequiredAvg: number;
+		spacingAvg: number;
+	};
+};
+
+export type PlaceDetailsWithAccesibilityData = {
+	result: Place;
+	accessibilityData?: {
+		_id: string;
+        guideDogAvg: number;
+		isMenuAccessibleAvg: number;
+		noiseLevelAvg: number;
+		lightingAvg: number;
+		isStaffHelpfulAvg: number;
+		isBathroomOnEntranceFloorAvg: number;
+		isContactlessPaymentOfferedAvg: number;
+		isStairsRequiredAvg: number;
+		spacingAvg: number;
+	};
```

Comment:
looks like this type is missing the promotions types from peer-server

Reply:
Sorry to make this confusing but idk if you looked at the main branch. I already made a push there and in the file you can see another type called 'Place' which has the promotion info. What you have highlighted here was taken from the frontend repo since these were some outdated types previously. Lmk if that makes sense!

Reply:
Also, I did not include 'PlaceWithA11yAndPromo' and 'PromotionMonth' which are types found in peer-server so let me know if you would like me to add those as well. I did not know if you guys want absolutely every type to be moved to the package or just types relevant to ratings.

Reply:
i think we should move any of those because they're reusable across the FE and BE - we don't need to move types like component prop types.

Also are the fields on this type a duplicate of the other Place type you're talking about, just without the promotion field?

Reply:
Ok sounds good.

I did not realize that they were basically duplicates! But yeah it seems they are duplicates without the promotion field. I guess we could just remove the ones without the promotion field then?

Reply:
I believe David will be helpful here since he knows the use of each type because I'm a little confused about the difference between some of them.

Reply:
>I guess we could just remove the ones without the promotion field then?
Yeah i think that's the way to go!